CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/clean/RemoveUnusedAttributesPluginNGTest.java

```diff
@@ -63,8 +65,10 @@ public void setUpMethod() {
      */
     @Test
     public void testAllNullAttributes() throws Exception {
-        PluginInteraction interaction = null;
-        PluginParameters parameters = null;
+
+        final PluginInteraction interaction = mock(PluginInteraction.class);
+        final PluginParameters parameters = mock(PluginParameters.class);
```

Comment:
for `interaction`, you may want to include some handling of the functions that will get called on the mock when the plugin is run else you may get some unintended behaviour.
for `parameters` I would be tempted to just leave it as null as given that parameter isn't use, there is not much value in setting it to anything

This might be similar for other tests (and possibly other test classes)